cmd/crio/main.go

```diff
@@ -547,9 +552,9 @@ func main() {
 			Usage:  "enable metrics endpoint for the server on localhost:9090",
 			EnvVar: "CONTAINER_ENABLE_METRICS",
 		},
-		cli.IntFlag{
+		cli.StringFlag{
```

Comment:
the need for this change is unclear to me. Wouldn't we want the verification of the port actually being an int? And toml can handle int values (`grpc_max_recv_msg_size` for instance)

Reply:
Okay, I will put it back to an int. The change was more for sake of consistency since every port we parse is of type string.

Reply:
ah, if that's the case I guess leaving it isn't problematic

Reply:
haha sorry, maybe revert it. keeping consistent and adding a verification step later makes sense to me.

Reply:
But now it might be possible that we can keep the command integration tests I removed before :man_shrugging:

Reply:
I'll leave it up to you :)

Reply:
I would leave it for now, I'd like your suggestion :heart: